docs/events.md

```diff
@@ -196,10 +148,18 @@ Event that represents a device sending its config to the services that are inter
     "id": "fbe64efa6c7f717e",
     "config": [{
       "sensorId": 1,
-      "change": true,
-      "timeSec": 10,
-      "lowerThreshold": 1000,
-      "upperThreshold": 3000
+      "schema": {
+        "typeId": 0xFFF1,
```

Comment:
What is this red background? Could it be removed?

Reply:
I believe this is from GitHub only 😄

Reply:
Kkkkkk

Reply:
I believe this is from GitHub only [2]